Mathlib/Data/Nat/Choose/Basic.lean

```diff
@@ -60,6 +60,21 @@ theorem choose_succ_succ (n k : ℕ) : choose (succ n) (succ k) = choose n k + c
 theorem choose_succ_succ' (n k : ℕ) : choose (n + 1) (k + 1) = choose n k + choose n (k + 1) :=
   rfl
 
+theorem choose_succ_left (n k : ℕ) (hk : 1 ≤ k) :
```

Comment:
Nitpick but I slightly prefer this (even though it is defeq)
```suggestion
theorem choose_succ_left (n k : ℕ) (hk : 0 < k) :
```
(e.g., not relevant here but you can only write `hk.ne` or `hk.ne'` with the `<` spelling)

Reply:
likewise below